scripts/Dockerfile.sim-install

```
@@ -0,0 +1,35 @@
+    ncurses-bin \
+    curl \
+    git \
+    ca-certificates \
+    gnupg \
+    && rm -rf /var/lib/apt/lists/*
+
+# Add the Raspberry Pi OS apt repo for the matching codename.
+# [trusted=yes] is a sim-only workaround: Debian Trixie's sqv rejects SHA1 keys
+# used by the raspi.com repo. Real Pi OS ships its own patched apt and does NOT
+# hit this issue.
+RUN echo "deb [trusted=yes] http://archive.raspberrypi.com/debian/ ${CODENAME} main" \
```

Hadolint (2.14.0)

[info] Delete the apt lists (/var/lib/apt/lists) after installing something

(DL3009)